### ISSUE TITLE
BatchNormalization problem

### ISSUE THREAD

**Issue by capineiro** [plaidml/plaidml#1653]
HI.
this error is displayed when i use the BatchNormalization layer

ERROR:plaidml:syntax error, unexpected ) : function (I) -> (O) { O = reshape(I, ); }
```
inputs = keras.layers.Input(shape=x_train.shape[1:])
drop1 = keras.layers.Dropout(0.1)(inputs)
bn1 =keras.layers.BatchNormalization()(drop1) 
conv1 = keras.layers.Conv2D(64, [8,16], activation='relu')(bn1)
```
plaidmi-keras 0.7.0
keras 2.2.4
